src/vt_flash/newton.jl

```diff
@@ -224,29 +272,36 @@ function vt_flash_newton!(
         newton_helmhess!,
         statex;
         constrain_step=constrain_step,
-        gtol=gtol,
+        gtol=NaN,
         maxiter=maxiter,
+        convcond=convcond,
     )
     statex .= optimresult.argument
 
     return __vt_flash_two_phase_result(state, mix, nmol, volume, RT, optimresult)
 end
 
 """
-    vt_flash_newton(mix, nmol, volume, RT)
+    vt_flash_newton(mix, nmol, volume, RT, StateVariables[; tol, chemtol, presstol, maxiter])
 
 Find VT-equilibrium of `mix`, at given `nmol`, `volume` and thermal energy `RT`
-using Newton's minimization. Return [`VTFlashResult`](@ref).
+using Newton's minimization.
+
+For the arguments see [`vt_flash`](@ref).
+
+Return [`VTFlashResult`](@ref).
 """
 function vt_flash_newton(
-    mix::BrusilovskyEoSMixture,
+    mix::BrusilovskyEoSMixture{T},
     nmol::AbstractVector,
     volume::Real,
     RT::Real,
     StateVariables::Type{<:AbstractVTFlashState};
-    gtol::Real=1e-3/RT,
+    tol::Real=1024*eps(T),
+    chemtol::Real=tol,
+    presstol::Real=tol,
     maxiter::Int=100,
-)
+) where {T}
```

Comment:
I left an opportunity to define `chemtol` and `presstol` separately, but, by default they are set to `tol`. Internally, `tol` isn't used, it's just for API.